README.md

```diff
@@ -1,37 +1,100 @@
 # Git Remotes + GitHub
 
+## Problem Statement
+
+GitHub does nothing special in the git universe. It's just another git
+repository in the cloud. Creating remote repositories on GitHub is very handy
+if you're ever working on a project while using multiple computers, and is
+especially helpful if you're working on a project with others. If you don't
+want to work with anyone else, you may not need remotes, but, in our case, we
+want to work with others! So, we have to talk about remotes.
+
 ## Objectives
-1. Describe GitHub and its relationship with Git
-2. Create a remote repository on GitHub
-3. Use `git push` to connect your local repository of files to your remote repository
-4. Use `git remote`
-5. Use `git push`
-6. Use `git pull`
 
-## Why this is useful
-GitHub does nothing special in the git universe. It's just another git repository in the cloud. If you don't want to work with anyone else, you don't need remotes. However, this is rarely the case, and we want to work with others! So, we have to talk about remotes.
+1. Explain creating a remote repository on GitHub
+2. Demonstrate connecting your local repository of files to your remote repository
+3. Use `git remote` to set the destination of your repo
+4. Use `git push` to send code to the remote repo
+5. Use `git pull` to retrieve code from the remote repo
+6. Use Github to make direct edits
+
 
-## Creating a remote repository on GitHub
-1. While logged into GitHub, click the :heavy_plus_sign: in the menubar and select `New repository`. Alternatively, just navigate to [github.com/new](https://github.com/new).
-2. Enter a name for your repository in the `Repository name` field. You can name it whatever you'd like; be creative! The default options are fine as-is — don't initialize the new repository with a README or add a `.gitignore` or license. Click the green `Create repository` button.
-3. After you create the repo, you should see a "Quick setup" page. Click the "Copy to clipboard" symbol next to the repo URL (pictured) to copy the URL. (We'll use this in the next section.)
+## Explain Creating a Remote Repository on GitHub
+
+1. While logged into GitHub, in the upper right corner, click the **+** in the
+menubar and select `New repository`. Alternatively, you can navigate to [github.com/new](https://github.com/new).
```

Comment:
Whoa. I'm half a mind to make this the path we use. This upper right kinda screen nav feels brittle. If the teacher shows + or they figure it out that's cool, but .... think about it.